swap_meet/item.py

```diff
@@ -1,2 +1,53 @@
+import uuid
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, id = None, condition = 0, age = 0):
+        # If no id is provided, generate a unique id using the uuid package 
+        # Instead of using uuid package to generate unique ids for Item instances,
+        #  I could use the built-in id function to generate unique ids based on the object's memory address. 
+        # This can simplify the code and avoid the dependency on an external package.
+        if id is None:
+            self.id = uuid.uuid4().int  # could be if use built-in function self.id = id(self) 
+        elif not isinstance(id, int):
+            raise ValueError("id must be an integer")
```

Comment:
Don't use `id(self)` as an id number; it's better to use `uuid` because that's actually unique across devices. `id(self)` isn't guaranteed, or even likely, to be unique.

A ternary is also fine here:

```python
        self.id = id if id else uuid4().int
```